### PR TITLE
70-quiz views

### PR DESCRIPTION
**Pull request by edpe**
https://trello.com/c/5Zi0xvOk/70-quiz-views-vvv-eee

### CODE REVIEW

File: pages/quiz/how-long-will-you-wait.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            <defs>
+              <clipPath id="wave" clipPathUnits="objectBoundingBox">
+                <path d="M 0,1  L 0,0  L 1,0  L 1,0.7  C .75 1.3, .25 .5, 0 1 Z" />
+              </clipPath>
+            </defs>
+          </svg>
+        </VisuallyHidden>
+        <Header>
+          <HeaderContents>
+            <Title>How long are you willing to wait?</Title>
```
Comment (Nikomus) [inline pages/quiz/how-long-will-you-wait.tsx:108]: We don't currently have any data that will be able to address this question. Are you able to make the pages match the ones listed in the card plan?

Comment (edpe) [reply pages/quiz/how-long-will-you-wait.tsx]: Sorry, I was following the Typeform design on that card, I'll remove the other pages.

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/filter**](https://my-best-life-pncne2ahq-neontribe.vercel.app/filter) | [**/**](https://my-best-life-pncne2ahq-neontribe.vercel.app/) |
|:-:|:-:|
|<a href="https://my-best-life-pncne2ahq-neontribe.vercel.app/filter">Screenshot of /filter</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/xH0Cq5MhiltCDCRqUieGH7xu9O0_.png">(view full size)</a> |<a href="https://my-best-life-pncne2ahq-neontribe.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/XexBh4mcoug6XFX4VO3hds2Ksqk_.png">(view full size)</a> |



| [**/quiz/about-you**](https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/about-you) | [**/quiz/helplines**](https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/helplines) |
|:-:|:-:|
|<a href="https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/about-you">Screenshot of /quiz/about-you</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/OwPOzlStfQnupnM6HJ5VPd_VoT4_.png">(view full size)</a> |<a href="https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/helplines">Screenshot of /quiz/helplines</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/ZqKbrxCrlglyRCLHzC2UYmAboZ8_.png">(view full size)</a> |



| [**/quiz/how-are-you-feeling**](https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/how-are-you-feeling) | [**/quiz/results**](https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/results) |
|:-:|:-:|
|<a href="https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/how-are-you-feeling">Screenshot of /quiz/how-are-you-feeling</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/jAuN386fPfG6uQRsLMvnk_rbX8s_.png">(view full size)</a> |<a href="https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/results">Screenshot of /quiz/results</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/nJwQNHWsdoXcTumawydDcPgw0kk_.png">(view full size)</a> |



And 3 other routes:
- [**/quiz/welcome**](https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/welcome)
- [**/quiz/what-are-your-interests**](https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/what-are-your-interests)
- [**/quiz/whats-on-your-mind**](https://my-best-life-pncne2ahq-neontribe.vercel.app/quiz/whats-on-your-mind)



Commit fb6e02014fd81e3c2ab4cf5e1159a92dc3e44b97 (https://my-best-life-pncne2ahq-neontribe.vercel.app).

**Comment by jthistle** [comment]
A few points:

- the `/quiz` route 404s (although, you can't get there via navigation)
- selecting any of the checkboxes selects all of them for some reason?

Screenshot from 2021-03-18 10-48-59

- a little bit of padding after the text might be good.

Apart from that, the basic structure all seems good.

**Comment by edpe** [comment]
> A few points:
> 
> * the `/quiz` route 404s (although, you can't get there via navigation)
> * selecting any of the checkboxes selects all of them for some reason?
> 
> Screenshot from 2021-03-18 10-48-59
> 
> * a little bit of padding after the text might be good.
> 
> Apart from that, the basic structure all seems good.



> A few points:
> 
> * the `/quiz` route 404s (although, you can't get there via navigation)
> * selecting any of the checkboxes selects all of them for some reason?
> 
> Screenshot from 2021-03-18 10-48-59
> 
> * a little bit of padding after the text might be good.
> 
> Apart from that, the basic structure all seems good.

Good shout - I'll change the quiz/welcome to index so you can go to /quiz

I've not done any styling yet, I just dumped  those in. The checkboxes are generated dynamically using our Checkbox component which needs some logic to work properly

**Review by jthistle — APPROVED** [review]
OK, this is all good then :)

File: src/Components/ImageCheckboxes.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  padding: 0 0.5rem;
+  margin: 0.5rem 0;
+
+  ${(props) => props.theme.screenSizes.tabletPortraitPlus} {
+    flex: 0 0 25%;
+  }
+
+  ${(props) => props.theme.screenSizes.desktopPlus} {
+    flex: 0 0 20%;
+  }
```
Comment (Nikomus) [inline src/Components/ImageCheckboxes.tsx:68]: These media queries will be expecting the page to be a lot wider than it can be here.

**Review by Nikomus — APPROVED** [review]
Let's add this in - we can look at the components in the next ticket

**Pull request merged**